html/index.d.ts

```diff
@@ -0,0 +1,1590 @@
+	| 'log'
+	| 'main'
+	| 'marquee'
+	| 'math'
+	| 'menu'
+	| 'menubar'
+	| 'menuitem'
+	| 'menuitemcheckbox'
+	| 'menuitemradio'
+	| 'navigation'
+	| 'none'
+	| 'note'
+	| 'option'
+	| 'presentation'
+	| 'progressbar'
+	| 'radio'
+	| 'radiogroup'
+	| 'region'
+	| 'row'
+	| 'rowgroup'
+	| 'rowheader'
+	| 'scrollbar'
+	| 'search'
+	| 'searchbox'
+	| 'separator'
+	| 'slider'
+	| 'spinbutton'
+	| 'status'
+	| 'switch'
+	| 'tab'
+	| 'table'
+	| 'tablist'
+	| 'tabpanel'
+	| 'term'
+	| 'textbox'
+	| 'timer'
+	| 'toolbar'
+	| 'tooltip'
+	| 'tree'
+	| 'treegrid'
+	| 'treeitem'
+	| (string & {});
+
+export interface HTMLAttributes<T extends EventTarget> extends AriaAttributes, DOMAttributes<T> {
+	// Standard HTML Attributes
+	accesskey?: string | undefined | null;
+	class?: string | undefined | null;
+	contenteditable?: Booleanish | "inherit" | undefined | null;
+	contextmenu?: string | undefined | null;
+	dir?: string | undefined | null;
+	draggable?: Booleanish | undefined | null;
+	hidden?: boolean | undefined | null;
+	id?: string | undefined | null;
+	lang?: string | undefined | null;
+	placeholder?: string | undefined | null;
+	slot?: string | undefined | null;
+	spellcheck?: Booleanish | undefined | null;
+	style?: string | undefined | null;
+	tabindex?: number | undefined | null;
+	title?: string | undefined | null;
+	translate?: "yes" | "no" | "" | undefined | null;
+	inert?: boolean | undefined | null;
+
+	// Unknown
+	radiogroup?: string | undefined | null; // <command>, <menuitem>
+
+	// WAI-ARIA
+	role?: AriaRole | undefined | null;
+
+	// RDFa Attributes
+	about?: string | undefined | null;
+	datatype?: string | undefined | null;
+	inlist?: any;
+	prefix?: string | undefined | null;
+	property?: string | undefined | null;
+	resource?: string | undefined | null;
+	typeof?: string | undefined | null;
+	vocab?: string | undefined | null;
+
+	// Non-standard Attributes
+	autocapitalize?: string | undefined | null;
+	autocorrect?: string | undefined | null;
+	autosave?: string | undefined | null;
+	color?: string | undefined | null;
+	itemprop?: string | undefined | null;
+	itemscope?: boolean | undefined | null;
+	itemtype?: string | undefined | null;
+	itemid?: string | undefined | null;
+	itemref?: string | undefined | null;
+	results?: number | undefined | null;
+	security?: string | undefined | null;
+	unselectable?: 'on' | 'off' | undefined | null;
+
+	// Living Standard
+	/**
+	 * Hints at the type of data that might be entered by the user while editing the element or its contents
+	 * @see https://html.spec.whatwg.org/multipage/interaction.html#input-modalities:-the-inputmode-attribute
+	 */
+	inputmode?: 'none' | 'text' | 'tel' | 'url' | 'email' | 'numeric' | 'decimal' | 'search' | undefined | null;
+	/**
+	 * Specify that a standard HTML element should behave like a defined custom built-in element
+	 * @see https://html.spec.whatwg.org/multipage/custom-elements.html#attr-is
+	 */
+	is?: string | undefined | null;
+
+	// Svelte bind:x
```

Comment:
Sorry for the super nits comment, but do we need this comment?

Reply:
I want to leave it in to make it more clear that the following attributes are not real DOM attributes, rather they are there so `bind:innerHTML` for example doesn't throw a type error.